### ISSUE TITLE
Drop downs are not WCAG compliant

### ISSUE THREAD

**Issue by amyjko** [wordplaydev/wordplay#330]
## Expected behavior

The menu should have a label that is read by the screen reader.

## Actual behavior

1. Enable a screen reader
2. Move keyboard focus to the select
3. The screen reader does not describe the purpose of the select.

**Comment by amyjko** [comment]
Generalized to all drop downs.
